Tools/TestWebKitAPI/Tests/WebKitCocoa/UserContentController.mm

```diff
@@ -1281,3 +1282,133 @@ - (void)userContentController:(WKUserContentController *)userContentController d
     [controller removeAllScriptMessageHandlers];
     EXPECT_NULL(weakHandler.get());
 }
+
+TEST(WKUserContentController, AllowAutofill)
+{
+    scriptMessagesVector.clear();
+    isDoneWithNavigation = false;
+    receivedScriptMessage = false;
+
+    auto options = adoptNS([[WKContentWorldOptions alloc] init]);
+    options.get().allowAutofill = YES;
+
+    RetainPtr<WKContentWorld> world = [WKContentWorld worldWithName:@"TestWorldAllowingAutofill" options:options.get()];
+    RetainPtr<ScriptMessageHandler> handler = adoptNS([[ScriptMessageHandler alloc] init]);
+    RetainPtr<WKUserScript> userScript = adoptNS([[WKUserScript alloc] _initWithSource:@"window.webkit.messageHandlers.testHandler.postMessage(typeof(document.createElement('input').autofillAvailable))"
```

Comment:
Can remove the `<type>` for all of these. And instances below.

Reply:
Good point. Removed.